cmd/tendermint/commands/run_node.go

```diff
@@ -79,6 +80,14 @@ func runNode(cmd *cobra.Command, args []string) error {
 		}
 	}
 
+	if !tmflags.IsLogLevelSimple(config.LogLevel) {
```

Comment:
why is this in run_node.go rather than integrated in root.go with the other parse logic?